src/query/api/v1/handler/prometheus/common.go

```diff
@@ -52,14 +53,15 @@ const (
 
 var (
 	matchValues = []byte(".*")
+	roleName    = []byte("role")
```

Comment:
hm this feels like a weird thing to modify a metric with. won't it conflict with people's tags?

Reply:
yeah, I've disabled stripping it for now

Prom does some dumb thing where it adds a "role":"remote" tag to each metric we get from the remote write endpoint which is questionable, but probably safer to leave it in.

Reply:
yeah, i'd avoid doing anything magicy to the metrics. can add down the road behind config if it's really required